tests/routes/pro/patch_user_test.py

```diff
@@ -64,6 +64,29 @@ def when_changes_are_allowed(self, app):
             "publicName": "publicName",
         }
 
+    @pytest.mark.usefixtures("db_session")
+    def should_remove_spaces_from_phone_number_before_db_save(self, app):
```

Comment:
```suggestion
    def should_remove_spaces_from_phone_number(self, app):
```
C'est un détail d'implémentation non ?